frontend/src/pages/SubmitProperty.tsx

```diff
@@ -152,13 +152,17 @@ const SubmitProperty = (props: any) => {
         </Row>
       </Col>
       <Col md={5} className="sideMap" title={parcelId ? '' : 'click on map to add a pin'}>
-        <MapView
-          disableMapFilterBar={true}
-          disabled={!!parcelId}
-          showParcelBoundaries={false}
-          onMarkerClick={() => {}}
-          onMarkerPopupClosed={() => {}}
-        />
+        {parcelDetailRequest?.isFetching || (parcelId && !cachedParcelDetail) ? (
```

Comment:
This  is the only change from the force-push. this makes this uses the same syntax as the other check on this page for consistency